roles/ansible-test-splitter/files/list_changed_targets.py

```diff
@@ -67,15 +67,20 @@ def read_collection_name(path):
         return f'{content["namespace"]}.{content["name"]}'
 
 
-def list_pyimport(prefix, module_content):
+def list_pyimport(prefix, subdir, module_content):
     root = ast.parse(module_content)
     for node in ast.walk(root):
-        if isinstance(node, ast.Import) and node.names[0].name.startswith(prefix):
+        if isinstance(node, ast.Import):
```

Comment:
Previously "prefix" was being overwritten, and when we resumed after a ImportFrom result would sometimes be an empty string rather than what it was supposed to be. 

Since we actually want to test against more than one collection, we technically still want to return even if the name doesn't start with prefix